monai/bundle/scripts.py

```diff
@@ -718,8 +720,8 @@ def run(
     if "config_file" not in _args:
         warnings.warn("`config_file` not provided for 'monai.bundle run'.")
     _log_input_summary(tag="run", args=_args)
-    config_file_, meta_file_, runner_id_, logging_file_, tracking_ = _pop_args(
-        _args, config_file=None, meta_file=None, runner_id="", logging_file=None, tracking=None
+    meta_file_, logging_file_, config_file_, runner_id_, tracking_ = _pop_args(
+        _args, "meta_file", "logging_file", config_file=None, runner_id="", tracking=None
```

Comment:
Hi @yiheng-wang-nv ,

I think we have 2 options here:
(1) Put the default values in the function definition directly as your PR, if so, please also put the `runner_id=""` to there.
(2) Put the default values in this `_pop_args` here, leave the function args as `default = None`.

I am OK with either way, what do you think?
BTW, please update the bundle run integration test to cover this change.

Thanks.

Reply:
Thanks Nic. I modified to use method 2 which is the same as `verify_metadata`